src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -4194,8 +4194,7 @@ def test_vmss_create_existing_options(self):
         ])
         self.cmd('network lb show --name {lb} -g {rg}',
                  checks=self.check('backendAddressPools[0].backendIPConfigurations[0].id.contains(@, \'{vmss}\')', True))
-        self.cmd('network vnet show --name {vnet} -g {rg}',
-                 checks=self.check('subnets[0].ipConfigurations[0].id.contains(@, \'{vmss}\')', True))
```

Comment:
Why do we need to remove this check?

Reply:
This keyword is converted to upper case in the response body and is not related to the version upgrading, so I remove it. I will add it back and change the check to case insensitive.